google-datacatalog-looker-connector/setup.py

```diff
@@ -37,7 +37,8 @@
         ],
     },
     include_package_data=True,
-    install_requires=('looker_sdk==0.1.3b7', 'google-datacatalog-connectors-commons>=0.5.2,<0.6.0'),
+    install_requires=('looker_sdk == 0.1.3b7',
+                      'google-datacatalog-connectors-commons >= 0.6.0'),
```

Comment:
Do you think we should change the `google-datacatalog-looker-connector` package version already? So it's clear that the `0.5.2` version uses the `google-datacatalog-connectors-commons` <`0.6.0` version.

The way it is, might be misleading.

Reply:
Sure, that makes sense. Done!